code/game/objects/items/devices/radio/headset.dm

```diff
@@ -290,6 +290,30 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 /obj/item/radio/headset/silicon/can_receive(freq, level)
 	return ..(freq, level, TRUE)
 
+/obj/item/radio/headset/clown
```

Comment:
Does clown not already get one? I swearthey have a pink and yellow headset already. Might need to be renamed ro be clearly not the clowns starting headset or use a component based approach instead

Reply:
![image](https://user-images.githubusercontent.com/73374039/183784912-f65427a0-352b-4c16-b144-7dc868467b10.png)
nope, just the service radio